_monitoring-plugins/alerting/api.md

```diff
@@ -508,17 +508,58 @@ POST _plugins/_alerting/monitors
+Per document monitors check whether or not individual documents in an index match trigger conditions to generate an alert notification. When you run a query with a per document monitor, the results are returned for each document that matches the trigger condition. You can create trigger conditions based on query names, query IDs, or tags that combine multiple queries. 
+
+The following table shows the syntax to use for each trigger option:
+
+Trigger options | Definition | Syntax
+:--- | :--- : :---
+Tag | Creates alerts for documents that match a muliple query with this tag applied. If you group multiple queries by a single tag, then you can set it to trigger an alert if the results are returned by this tag name.| `query[tag=<tag-name>]`
+Query by name | Creates alerts for documents matched or returned by the named query.  | `query[name=<query-name>]`
+Query by ID | Creates alerts for documents that were returned by the identified query. | `query[id=<query-id>]`
+
+For more information about per document monitors, see [Monitor types]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/monitors/#monitor-types).
+
+### Search for monitor findings
+
+You can search the findings index `.opensearch-alerting-finding*` for available document findings with a GET request to the findings resource:
 
-You can combine multiple queries in a per document monitor by adding a tag to each query and then setting the trigger condition to that tag.
+```json
+GET /_plugins/_alerting/findings/_search?
+```
+The response returns all findings available.
+
+To retrieve metadata for an individual document finding entry, you can search for the finding by its `findingId` as follows:
+
+```json
+GET /_plugins/_alerting/findings/_search?findingId=gKQhj8WJit3BxjGfiOXC
+```
 
-For more information about document-level monitors, see [Monitor types]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/monitors/#monitor-types).
+The response returns the number of individual finding entries in the `total_findings` field.
 
-#### Sample request
+To get more specific results in a findings search, you can use the path parameters defined in the following table:
+
+Path parameter | Description | Usage
+:--- | :--- : :---
+`findingId` | The identifier for the finding entry. | The finding ID is returned in the initial query response.
+`sortString` | This field sorts the finding. | The default value is `id`.
+`sortOrder` | The order to sort the list of findings, either ascending or descending. | Use `sortOrder=asc` to indicate ascending, or `sortOrder=desc` for descending sort order.
+`size` | The maximum number of results to retrieve. | Can be any integer.
+`startIndex` | The pagination indicator. | Default is `0`.
+`searchString` | The finding attribute you want returned in the search. | You might want to search findings for a specific index, provide `searchString=indexABC'.
```

Comment:
"To search in a specific index, specify the index's name in your request's path. For example, ..."